doc/developers/bug_triaging.rst

```diff
@@ -148,5 +150,21 @@ The following workflow [*]_ is a good way to approach issue triaging:
    An additional useful step can be to tag the corresponding module e.g.
    `sklearn.linear_models` when relevant.
 
-.. [*] Adapted from the pandas project `maintainers guide
+Becoming a member of the scikit-learn triage team
+--------------------------------------------------
+
+Any contributor may become a member of the scikit-learn triage team, after
+showing some continuity in participating to scikit-learn
+development (with pull requests and reviews).
+Any core developer are welcome to propose a scikit-learn contributor to the
+triage team [2]_. Without objections from other core developers, the
```

Comment:
should this ref be made about the first sentence? i.e. 

`Any contributor may become a member of the scikit-learn triage team [2]_`

Reply:
Well technically the first sentence is not quoted from the python page, while the second is... Do you mind if I put the note at the end of the paragraph then, to precise that the entire paragraph is inspired by?